descartes_trajectory/test/cart_trajectory_pt.cpp

```diff
@@ -1,160 +0,0 @@
-/*
```

Comment:
It doesn't look like all the tests in this file got captured in the new file above?